### DISCUSSION TITLE
機械翻訳の誤訳

### DISCUSSION THREAD

**Discussion by tatsuo-ishii** [pgsql-jp/jpug-doc#2620] [General]
### 重要度

高

### 問題点

「みんなの自動翻訳」による翻訳で、"more than two"が「2つ以上」と訳されることに気が付きました。同様に、"more than three"は「3つ以上」と訳されてしまいます。

### 背景

誤訳だと思うのですが、NICTはそうは思っていないようです。
https://mt-auto-minhon-mlt.ucri.jgn-x.jp/content/qa/detail/?q_qa_id=1995&t=1669961350

### 注意点

というわけで、直る見込みがないので、皆さん気をつけましょう、という注意喚起です。

### 貢献者として記載可否

記載（貢献者欄に書いてください）

**Comment by noborus** [comment]
more thanはDeepLや他の機械翻訳でも同じような訳になるので難しそうですね。
直る見込みはないかと思ってたのですが、チューニングを試しているらしく、ちょっと可能性出てきてますね。

**Comment by tatsuo-ishii** [comment]
> more thanはDeepLや他の機械翻訳でも同じような訳になるので難しそうですね。

more than oneは「複数の」となるのでOKなのですが、two以上だと駄目なんですよね。

みんなの自動翻訳は調整いただいているので、とりあえず解決済みとしておきました。

**Comment by noborus** [comment]
これはクローズするよりもDisscussionに移動して、なんか出てきたら書き足すのが良いと思いますので移動しますね。
